Mathlib/Analysis/SpecificLimits/Basic.lean

```diff
@@ -110,6 +110,19 @@ theorem tendsto_pow_atTop_nhds_0_of_lt_1 {𝕜 : Type _} [LinearOrderedField 
       (tendsto_inv_atTop_zero.comp this).congr fun n => by simp)
 #align tendsto_pow_at_top_nhds_0_of_lt_1 tendsto_pow_atTop_nhds_0_of_lt_1
 
+theorem tendsto_pow_atTop_nhds_0_iff_lt_1 {𝕜 : Type _} [LinearOrderedField 𝕜] [Archimedean 𝕜]
+    [TopologicalSpace 𝕜] [OrderTopology 𝕜] {r : 𝕜} (h₁ : 0 ≤ r) :
+    r < 1 ↔ Tendsto (fun n : ℕ => r ^ n) atTop (𝓝 0) := by
```

Comment:
I think a more useful version would be:
```lean
@[simp]
theorem tendsto_pow_atTop_nhds_zero_iff_abs_lt_one
    {𝕜 : Type _} [LinearOrderedField 𝕜] [Archimedean 𝕜]
    [TopologicalSpace 𝕜] [OrderTopology 𝕜] {r : 𝕜} :
    Tendsto (fun n : ℕ => r ^ n) atTop (𝓝 0) ↔ |r| < 1 := by
```
Hopefully this should be easy to prove by combining what you've got with `tendsto_zero_iff_abs_tendsto_zero`.